assets/javascripts/templates/pages/about_tmpl.coffee

```diff
@@ -626,6 +626,11 @@ credits = [
     '2005-2022 NumPy Developers',
     'BSD',
     'https://raw.githubusercontent.com/numpy/numpy/master/LICENSE.txt'
+  ], [
+    'NuxtJS',
+    '2016-present Nuxt Team',
+    'MIT',
+    'https://raw.githubusercontent.com/nuxt/nuxt.js/dev/LICENSE'
```

Comment:
```suggestion
    'https://raw.githubusercontent.com/nuxt/nuxtjs.org/main/LICENSE.md'
```

Here, we display the license of the docs not the source code.

This one uses https://creativecommons.org/licenses/by-nd/4.0/ "Creative Commons Attribution-NoDerivatives 4.0" – is devdocs.io a derivative? Rather yes? https://creativecommons.org/faq/#what-is-an-adaptation 

Here's how we dealt with it in the past https://github.com/freeCodeCamp/devdocs/issues/1752#issuecomment-1128620626

Reply:
Will try reach out to them to get some clarification. But yeah seems this would be a blocker

Reply:
I have made some updates to the mc-stan issue regarding the no derivatives clause of the CC license.